stable/kubecost/values.yaml

```diff
@@ -63,6 +63,8 @@ cost-analyzer:
     serviceAccounts:
       nodeExporter:
         create: false
+    kube-state-metrics:
+      priorityClassName: dkp-high-priority
```

Comment:
do you think its better to define these in the k-apps repo to align with the other applications? it might be easier for users to override if they see the priority classes directly in the default configmaps 🤔

Reply:
yeah that's a good reason.. my reasoning here was just because this is a parent chart that is basically kommander-specific but that's the only reason why really. I'll move it into the default value overrides

Reply:
reverted